src/sentry/static/sentry/app/views/issueList/overview.jsx

```diff
@@ -389,7 +389,7 @@ const IssueList = createReactClass({
     }
   },
 
-  onRealtimePoll(data, links) {
+  onRealtimePoll(data, _links) {
```

Comment:
i'm assuming this is because a linter is complaining but you don't wanna use just `_` for documentation purposes?